src/io/writer_internal.cpp

```diff
@@ -1,25 +1,35 @@
 #include <cstring>
+#include <stdexcept>
 
 #include <lazperf/filestream.hpp>
 #include <lazperf/lazperf.hpp>
 
+#include <copc-lib/copc/extents.hpp>
 #include <copc-lib/io/internal/writer_internal.hpp>
 #include <copc-lib/las/utils.hpp>
 #include <copc-lib/laz/compressor.hpp>
 
-using namespace lazperf;
-
 namespace copc::Internal
 {
 
 WriterInternal::WriterInternal(std::ostream &out_stream, const std::shared_ptr<CopcFile> &file,
                                std::shared_ptr<Hierarchy> hierarchy)
     : out_stream_(out_stream), file_(file), hierarchy_(hierarchy)
 {
-    // are extra bytes allowed to be an EVLR? If so we should just move it to be an EVLR
-    // but I don't know...
+    size_t extents_offset = CopcExtents(file_->GetLasHeader().point_format_id, file_->GetExtraBytes().items.size())
+                                .ToCopcExtentsVlr()
+                                .size() +
```

Comment:
should this be a static function of CopcExtents?

`def GetByteSize(point_format_id, eb_size) : size_t`